ideascube/monitoring/tests/test_views.py

```diff
@@ -216,6 +226,16 @@ def test_staff_can_create_specimen_with_letters_and_ints_in_barcode(staffapp):
     assert item.specimens.filter(barcode='123abc').count()
 
 
+def test_staff_can_create_specimen_with_letters_and_ints_in_serial(staffapp):
```

Comment:
> Functionally, can a serial contain letters and ints?

I see no reason why it couldn't.

> If so, how is this related to this commit?

I've used the same tests that for the specimen creation with barcode.